### ISSUE TITLE
Average logits could be saved inside this function call

### ISSUE THREAD

**Issue by Bala93** [Bala93/MarginLoss#3]
https://github.com/Bala93/MarginLoss/blob/7ded9ff653ac7441133d211aa7f89e7f135311a6/calibrate/engine/tester.py#L96
